src/plugins/elasticsearch/index.js

```diff
@@ -20,6 +22,7 @@ module.exports = function ({ Plugin }) {
         password: string(),
         shardTimeout: number().default(0),
         requestTimeout: number().default(30000),
+        requestHeadersWhitelist: array().items().single().default(DEFAULT_REQUEST_HEADERS),
```

Comment:
It allows the user to define a single item array as a bare string. So they can define the setting as:

   elasticsearch.requestHeadersWhiteList: x-my-custom-header-1

instead of having to say:

   elasticsearch.requestHeadersWhiteList: [ x-my-custom-header-1 ]

`.single()` will convert `x-my-custom-header-1` to `[ x-my-custom-header-1 ]` because it knows the value of this setting must ultimately be an array.